relayer/relays/beacon/config/config.go

```diff
@@ -56,10 +58,20 @@ type ActiveSpec string
 const (
 	Mainnet ActiveSpec = "mainnet"
 	Minimal ActiveSpec = "minimal"
+	GOERLI  ActiveSpec = "goerli"
+)
+
+const (
+	Minimal_CapellaForkEpoch uint64 = 0
+	Goerli_CapellaForkEpoch  uint64 = 162304
+	// TODO: change when timeline of Capella upgrade in mainnet finalized
+	Mainnet_CapellaForkEpoch uint64 = math.MaxUint64
 )
 
 func (c Config) GetActiveSpec() ActiveSpec {
 	switch c.Source.Beacon.ActiveSpec {
+	case string(GOERLI):
+		return GOERLI
```

Comment:
I'm also wondering why we need to add Goerli as a spec if Goerli's network config is the same as mainnet?

Reply:
sorry I missed your comment [before](`https://github.com/Snowfork/snowbridge/pull/786#discussion_r1131957837`).
Yes, we can find a better way to config `ForkingEpoch` for respective network without polluting `ActiveSpec` here.

Reply:
https://github.com/Snowfork/snowbridge/pull/786/commits/39a9d7ede9f7cfa25afbc848042da08cf98dc783